addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -107,13 +107,14 @@
                                 </strong>
                             </button>
                         </div>
+                        <div attrs="{'invisible': [('state', '!=', 'in_queue')]}" role="alert">
```

Comment:
Do you need `role="alert"` here? Doesn't seem to be on other DIVs.
A part from that, looking good!

Reply:
You right, I forgot to remove it.
PR updated